### PR TITLE
Insurance nav items

### PR DESCRIPTION
**Pull request by darkodosenovic**
* Adding navigation for Insurance-oa app
* Increasing version as a minor change

### CODE REVIEW

File: lib/ama_layout/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module AmaLayout
-  VERSION = "1.1.21"
+  VERSION = "1.3.0"
```
Comment (adamrmelnyk) [inline lib/ama_layout/version.rb]: Should this be going from `1.1.21` to `1.3.0` or should it be going to `1.1.22` or `1.2.0`
Should we just decide right now how this gem should be versioned?

File: lib/ama_layout/navigation.yml [hunk 2]
```diff
@@ -18,6 +18,14 @@ member:
   - text: "Driver Education"
     link: "<%= Rails.configuration.driveredonline_site %>"
     icon: "fa-car"
+  - text: "Insurance"
+    link: "<%= Rails.configuration.insurance_site %>/"
```
Comment (adamrmelnyk) [inline lib/ama_layout/navigation.yml]: This is already in the ENV's

**Comment by adamrmelnyk** [comment]
:hankey:

File: lib/ama_layout/version.rb (hunk 1 above)
Comment (darkodosenovic) [inline lib/ama_layout/version.rb]: My bad . It should be 1.2.0. Minor change when adding a new nav items.

**Comment by darkodosenovic** [comment]
:wave:

File: lib/ama_layout/navigation.yml [hunk 3]
```diff
@@ -18,6 +18,14 @@ member:
...
+    icon: "fa-umbrella"
+    sub_nav:
+      - text: "Insurance Overview"
+        link: "<%= Rails.configuration.insurance_site %>/"
```
Comment (MathieuGilbert) [inline lib/ama_layout/navigation.yml]: Should this be using the helper path?

File: lib/ama_layout/navigation.yml [hunk 4]
```diff
@@ -18,6 +18,14 @@ member:
...
+      - text: "View My Policies"
+        link: "<%= Rails.configuration.insurance_site %>/policies"
```
Comment (MathieuGilbert) [inline lib/ama_layout/navigation.yml]: Should this be using the helper path?

**Comment by MathieuGilbert** [comment]
:hankey: Those helper paths look like they should be used somewhere.

**Comment by darkodosenovic** [comment]
:wave:

**Comment by weyermannx** [comment]
:elephant: :elephant: :elephant:

**Comment by adamrmelnyk** [comment]
:hankey: needs master

**Comment by darkodosenovic** [comment]
:wave:

**Comment by MathieuGilbert** [comment]
:hankey: Out of date again...

**Comment by darkodosenovic** [comment]
Yeah, I am not going to bother to update this until is ready to go live. So :hankey: for now

**Comment by jbabe** [comment]
:hankey: 
we need to set this up to run on Travis CI

**Pull request closed without merging**